view/partial/add-trial.hbs

```diff
@@ -63,3 +75,7 @@
         </div>
     </div>
 </div>
+$( "input[name='stagecount']" ).change(function() {
```

Comment:
it would be fantastic if the JavaScript here could be moved to a new file `view/script/add-patient.js`.
and then it could be loading into the page using
``` html
<script src="/static/script/add-patient.js"></script>
```